wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -743,24 +775,39 @@ def run(
                 )
             except ApiException as e:
                 raise LaunchError(
-                    f"Error creating CRD of kind {kind}: {e.status} {e.reason}"
+                    f"Error creating CRD of kind {kind}: {e.status} {e.reason} {e.body} {e.headers}"
```

Comment:
I ripped out the header and am going to display the body starting on the next line as clean yaml. This is actually really useful so that users trying to launch CRDs can see the error in their config